tests/conftest.py

```diff
@@ -1,5 +1,4 @@
 # setup for py.test
```

Comment:
We can remove this useless comment since we're in here, `conftest` inherently sets up pytest.